tools/grama.sed

```diff
@@ -1,6 +0,0 @@
-# remove index "see" entries
-/\\index.*|see/d
```

Comment:
That's right, this construction doesn't exist at the moment. (You can see what's extracted in `std-gram.ext`; and there are no PDF diffs from this PR.) In the future, if such a situation comes up, we can perhaps work around with with `extract` and `extractskip` environments, or we can build another special-purpose command that changes meaning in the grammar.